packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -30,3 +30,7 @@ export function showCLINotInstalledMessage() {
   );
   window.showWarningMessage(showMessage);
 }
+
+export function isSFDXContainerMode(): boolean {
+  return process.env.SFDX_CONTAINER_MODE === 'true';
```

Comment:
Are we doing this instead of evaluating `context.executionContext === vscode.ExtensionExecutionContext.Remote` when activating the extension, like it's mentioned in the [Supporting Remote Development docs](https://code.visualstudio.com/api/advanced-topics/remote-extensions#using-native-node.js-modules), because it's still a proposed/unstable api ?

Is this being set when the user spins up the container ?

Reply:
Yes. That new setting requires VS Code API 1.36 (latest). I’ve asked Jonny to abstract the check into the util here so that we can update and use that new setting later when it’s ready. For now we can just use the ENV var.

Reply:
@vazexqi I check for the value of 'true' because I thought it would be weird if it was just any value, for example `SFDX_CONTAINER_MODE = 'false'` or has a value of `"this is not in a container"`
I guess someone would have to override the value for some reason but I figured why not make it `true`.
Just my thought process, I'm open to simply 
``` return process.env.SFDX_CONTAINER_MODE```

Reply:
This is fine for now.

Reply:
I looked at the way that we have been using ENV variables in the CLI (https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_cli_env_variables.htm).

What you have done (checking for `true`) is consistent with what the CLI is doing, so no changes now (or in the future) are necessary.

Just for context, this is not usually how other CLIs do it. Some other ones just need the ENV to be defined.

Reply:
ok, thanks for checking on that.